source/Playnite/SortableNameConverter.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System;
```

Comment:
Please try to follow some sensible ordering of class members. Like all private fields/properties, then public, then constructor then methods. It's mixed all together now.